students/kyungchulpark/user/views.py

```diff
@@ -1 +1,34 @@
+import json
 
+from django.http  import JsonResponse
+from django.views import View
+from .models      import User
+from .validators  import validate_phone,validate_email,validate_password,validate_nicname
+
+class UserView(View):
+    def post(self,request):
+        try:
+            data = json.loads(request.body)
+        
+            email_data    = data['email'] 
+            password_data = data['password'] 
+            nicname_data  = data['nicname'] 
+            phone_data    = data['phone'] 
+
+            validate_email(email_data)
+            validate_password(password_data)
+            validate_phone(phone_data)
+            validate_nicname(nicname_data)
+
+            User.objects.create(
+                email=email_data,
+                password=password_data,
+                nicname=nicname_data,
+                phone=phone_data)
```

Comment:
아래와 같이 깔끔하게 정렬될 수 있도록 신경써주세요!
```suggestion
            User.objects.create(
                email    = email_data,
                password = password_data,
                nicname  = nicname_data,
                phone    = phone_data
            )
```